src/Resources/views/stack.html.twig

```diff
@@ -38,7 +38,7 @@
             {{ stack.clientRequest|httplug_markup|nl2br }}
         </div>
         <div class="httplug-message card">
-            <h4>Response</h4>
+            <h4>Response {% if stack.debugTokenLink %}<span class="label status-info"><a href="{{ stack.debugTokenLink }}">{{ stack.debugTokenLink }}</a></span>{% endif %}</h4>
```

Comment:
Possibly instead of showing the full link it might make sense to only display the token. In that case I guess an icon or something else indicating what the token is for would be helpful. Maybe there are other suggestions?

Reply:
i like the idea of showing e.g. the token symfony uses itself for the debug toolbar, along with a "external link" icon. showing the full url could be quite long, and shortening might not show the right fragment.

Reply:
I replaced the URL with the Token (see screenshot). I decided against adding an icon for now. If there is an svg I could use where we know that licensing is not an issue we can add it later.

Reply:
awesome! as there is no super obvious icon that all of us immediately think of, i prefer text over icon.